### PR TITLE
Proposal: Add byline renamer for commonly misspelled bylines

### PR DESCRIPTION
**Pull request by currysoup**
Turns out or good pal Mat has been maintaining a list of commonly misspelled and manually fixing them.

Shall we wrap this into a metadata cleaner?

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/PhotographerRenamer.scala [hunk 1]
```diff
@@ -0,0 +1,140 @@
+package com.gu.mediaservice.lib.cleanup
+
+import com.gu.mediaservice.model.ImageMetadata
+
+object PhotographerRenamer extends MetadataCleaner {
```
Comment (franziskas) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/PhotographerRenamer.scala]: I can haz 1 test plz???

**Comment by paperboyo** [comment]
💎!

**Comment by currysoup** [comment]
Yeah! If it was cached locally for performance and had a little page for adding new ones that would be so neat!

**Comment by paperboyo** [comment]
💯 to both comments. Most cleaners should be in the hands of (power)users TBH. Benefits would be mutual ;-)

Until then, expect me to be adding to this list regularly…

**Pull request merged**